docs/Proposal guide.md

```diff
@@ -0,0 +1,153 @@
+#### Introductory note
+This guide can be used by Flock program applicants in order to draft their ideal proposal. However, the structure and content of the proposal can be adapted as much as necessary by the applicant team. Above all, applications should be **clear and concise**.
+
+Don't forget to share the proposal as early as possible with the [community](https://forum.aragon.org/).
+
+# Aragon Flock proposal: Team name
+
+## 1. Summary
+
+### Strategy
+*This section should detail the high level strategy and rational behind the proposal in simple terms. It can be accompanied by a few short references to past team achievements.*
```

Comment:
```suggestion
*This section should detail the high level strategy and rationale behind the proposal in simple terms. It can be accompanied by a few short references to past team achievements.*
```